lib/google_scraping/account/helpers/user_token.ex

```diff
@@ -0,0 +1,39 @@
+defmodule GoogleScraping.Accounts.UserToken do
```

Comment:
As this is the schema file, how about to follow our schema folder structure? like `GoogleScraping.Account.Schemas.UserToken`?

![Screen Shot 2022-05-23 at 3 40 20 PM](https://user-images.githubusercontent.com/6076762/169779566-082f3e31-e884-46ee-8f29-72f09041a937.png)

Reply:
thank you for pointing on this option!